lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/TransactionReceipt.kt

```diff
@@ -83,19 +93,19 @@ data class InvokeTransactionReceipt(
     override val actualFee: Felt,
 
     @JsonNames("block_hash")
-    override val blockHash: Felt,
+    override val blockHash: Felt? = null,
```

Comment:
I'd add FIXME comment to this indicating why this is not following RPC spec currently, and create an issue for updating these once rpc spec get resolved.